docs/DeveloperGuide.md

```diff
@@ -303,32 +306,68 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
-* 2a. The list is empty.
+- 2a. The list is empty.
 
   Use case ends.
 
-* 3a. The given index is invalid.
+- 3a. The given index is invalid.
+
+  - 3a1. AddressBook shows an error message.
 
-    * 3a1. AddressBook shows an error message.
+    Use case resumes at step 2.
 
-      Use case resumes at step 2.
+**Use case: UC2 Load contact from save file**
 
-*{More to be added}*
+**Actor: BizBook**
+
+**Guarantees:** Contact list is initialized with available data.
+
+**MSS**
+
+1.  BizBook searches OS directory for save file.
+2.  OS returns save file object.
+3.  For each line is save file.
+
+    - 3.1 BizBook reads a line from the file.
+    - 3.2 BizBook parses the line into a contact object.
+    - 3.3 BizBook adds the contact object to the contact list.
+    - 3.4 Repeat steps 3.1 - 3.3 until all lines are processed.
+
+4.  BizBook finishes loading contacts into the contact list.
+
+    Use case ends.
+
+**Extensions**
+
+- 1a. Save file does not exist.
+
+  - 1a1. BizBook shows a warning message.
```

Comment:
I think it may be better to use `BizBook` for all instances of BizBook for a nicer formatting